builder/tests/builder/src/test/java/io/helidon/builder/test/CustomNamedTest.java

```diff
@@ -32,24 +31,36 @@ class CustomNamedTest {
 
     @Test
     void testIt() throws Exception {
-        DefaultCustomNamed.Builder customNamedBuilder = DefaultCustomNamed.builder()
-                .addStringList("b").addStringList("a").addStringList("b").addStringList("y")
-                .addStringToIntegerMap("b", 1).addStringToIntegerMap("e",2).addStringToIntegerMap("a", 3)
-                .addStringSet("b").addStringSet("a").addStringSet("b").addStringSet("y");
+        CustomNamed.Builder customNamedBuilder = CustomNamed.builder()
+                .addStringList("b")
+                .addStringList("a")
+                .addStringList("b")
+                .addStringList("y")
+                .putStringToIntegerMap("b", 1)
+                .putStringToIntegerMap("e", 2)
+                .putStringToIntegerMap("a", 3)
+                .addStringSet("b")
+                .addStringSet("a")
+                .addStringSet("b")
+                .addStringSet("y");
+
         CustomNamed customNamed = customNamedBuilder.build();
-        assertThat("should be ordered since we are using tree and ordered/linked",
+
+        // no longer supported, only linked values are used (order of insertion)
+        //        assertThat("should be ordered since we are using tree and ordered/linked",
+        //                   customNamed.toString(),
+        //                   equalTo("CustomNamed(stringSet=[a, b, y], stringList=[b, a, b, y], stringToIntegerMap={a=3, b=1, "
+        //                                   + "e=2})"));
+
+        assertThat("should be ordered since we are using linked types",
                    customNamed.toString(),
-                   equalTo("CustomNamed(stringSet=[a, b, y], stringList=[b, a, b, y], stringToIntegerMap={a=3, b=1, "
-                                   + "e=2})"));
+                   equalTo("CustomNamed{stringSet=[b, a, y],stringList=[b, a, b, y],stringToIntegerMap={b=1, e=2, a=3}}"));
```

Comment:
do we have a `Confidential` test anywhere?  We should ensure it is not printed in any use - String and char[]

Reply:
added a follow up